tests/helpers/unit.js

```diff
@@ -90,6 +90,10 @@ helpers.unit = function(method, error) {
           stub.db = helpers.couch + stub.db;
         }
 
+        if (method[1] === 'follow' || method[1] === 'followUpdates') {
+          delete req.httpAgent; // ignore 'httpAgent' in deep equal assert
```

Comment:
So if the `httpAgent` is getting passed through from `nano` to follow shouldn't it always be the same `httpAgent`?
Or do the tests here use multiple nano instances to assert?

Reply:
As per discussion I see the assertion is just against a mock `{db: ...}` object so this makes sense.